hadoop-project/pom.xml

```diff
@@ -220,62 +218,6 @@
 
   <dependencyManagement>
     <dependencies>
-      <dependency>
```

Comment:
this should be reinstated with the new exclusions; then the test declarations don't need changing.

Reply:
I've committed some changes to bring back some of dependency management to this pom

Reply:
thanks